Warp.WebApp/Controllers/EntryController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System.Threading;
+using Warp.WebApp.Helpers;
+using Warp.WebApp.Services.User;
+using Warp.WebApp.Services;
+using Warp.WebApp.Services.Entries;
+
+namespace Warp.WebApp.Controllers;
+
+[ApiController]
+[Route("/api/entry")]
+public class EntryController : BaseController
+{
+    public EntryController(IEntryService entryService)
+    {
+        _entryService = entryService;
+    }
+
+
+    [HttpDelete("delete")]
```

Comment:
No need to use a name for the method. `[HttpDelete()]` is enough, because we need no additional details for the action